### PR TITLE
Ability to pass id prop to whole table

### PR DESCRIPTION
**Pull request by KamilRybacki**
Hello!

Thanks for Your awesome library! :+1: 
I've been working with it lately and noticed that there wasn't a way to assing a global `id` attribute via `props` to the whole table, since it wasn't being deconstructed within `SimpleTable<Ind extends IndexType = IndexType>(props: Props<Ind>)` function in `SimpleTable.tsx`.

Therefore, I propose adding this feature since it is always awesome to be able to query things via unique `id`s e.g. during tests of DOM manipulations :smile: 

Lemme know if You agree with my idea and/or styling of this new section of code.

Best regards :pray:

### CODE REVIEW

**Review by aminya — CHANGES_REQUESTED** [review]
It would be great if you could add a test

**Comment by KamilRybacki** [comment]
Hi! Thanks for replying, I've seem to have missed You comment due to some mishap in notifications. I have added the tests You have mentions, together with changes to demo tables & SimpleTable component, so that the id of the rendered elements id correctly passed thru props.

**Review by aminya — APPROVED** [review]
Thank you for adding the tests!

File: src/SimpleTable.tsx [hunk 1]
```diff
@@ -82,7 +83,11 @@ export function SimpleTable<Ind extends IndexType = IndexType>(props: Props<Ind>
   sortRows()
 
   return (
-    <table class={props.className ?? "solid-simple-table light typography"} style={props.style}>
+    <table
+      className={props.className ?? "solid-simple-table light typography"}
```
Comment (aminya) [inline src/SimpleTable.tsx:87]:
```suggestion
      class={props.className ?? "solid-simple-table light typography"}
```

**Pull request merged**